happi/client.py

```diff
@@ -325,6 +326,84 @@ def load_device(self, use_cache=True, **post):
         cntr = self.find_device(**post)
         return from_container(cntr, use_cache=use_cache)
 
+    def change_container(
+        self,
+        item: HappiItem,
+        target: Type[HappiItem],
+        edits: Dict[str, Any] = {},
+        how: str = 'right'
+    ) -> Dict[str, Any]:
+        """
+        Takes an instance of one item (device) and transfers its contents
+        into a new ``target`` container.  Checks are performed to ensure
+        the contents are compatible (follwing enforce requirements in the
+        target container)
+
+        This function is built to be a helper function for an interactive
+        cli tool, and passes information up accordingly.  If keys are not
+        significantly mismatched, this function can be used as is.
+
+        Parameters
+        ----------
+        item : happi.HappiItem
+            HappiItem instance to be transferred to a new container
+        target : Type[happi.HappiItem]
+            Container to contain new item
+        edits : dict[str, Any], optional
+            Dictionary of edits to supercede values in original item
```

Comment:
```suggestion
            Dictionary of edits to supersede values in original item
```

I was convinced it was spelled this way for the longest time...

Reply:
TIL, I always thought it was from the same root as "cede" (latin/french, cedere/ceder, to yield), not from "sede" (latin, sedere, to sit)

Reply:
Every day I think "I should put a spell checker in my ide" and then I remember my entire file would be covered in red squiggles.